src/Raven.Server/Web/System/AdminDatabasesHandler.cs

```diff
@@ -1242,15 +1233,20 @@ public async Task MigrateDatabaseOffline()
             }
 
             var dataDir = configuration.DataDirectory;
-            OfflineMigrationConfiguration.ValidateDataDirectory(dataDir);
+            var dataDirectoryThatWillBeUsed =  string.IsNullOrWhiteSpace(dataDir) ? 
+                                               ServerStore.Configuration.Core.DataDirectory.FullPath :
+                                               PathUtil.ToFullPath(dataDir, ServerStore.Configuration.Core.DataDirectory.FullPath); 
```

Comment:
I think it would be better to use `PathSetting` it performs this + additional validation

Reply:
done